### PR TITLE
update in plotting and bug fix

### PR DESCRIPTION
**Pull request by licode**
This is still in process, and should be done soon.

### CODE REVIEW

**Comment by arkilic** [comment]
Is this good to go @licode ?

**Comment by licode** [comment]
I need to double check it, and let you know.

File: pyxrf/model/fileio.py [hunk 1]
```diff
@@ -1697,6 +1697,18 @@ def _make_hdf(fpath, runid):
...
+
+        # use suitcase to save baseline data, and scaler data from primary
+        tmp = set()
+        for descriptor in hdr.descriptors:
+            xs3 = [key for key in descriptor.data_keys.keys() if 'xspress3' in key]
+            roi = [key for key in descriptor.data_keys.keys() if 'Det' in key]
+            tmp.update(xs3)
+            tmp.update(roi)
+            tmp.add('merlin1')
+            print(tmp)
```
Comment (arkilic) [inline pyxrf/model/fileio.py]: Will you add some sort of description to your print in addition to contents of set to make it more explicit?

Comment (licode) [inline pyxrf/model/fileio.py]: thanks for comments. This is just to test. I will remove it in a cleaner version. And not ready to merge yet.

**Pull request merged**